app/src/main/res/values/strings.xml

```diff
@@ -56,4 +57,30 @@
     <string name="order_range">Order Range</string>
     <string name="tickets_available">Tickets Available</string>
 
+    <!--suggestion mail details-->
+    <string name="emailInfo">List out your proposed Bug(s)/Feature(s)</string>
+    <string name="emailSubject">Improvement Suggestion for Open Event General App</string>
+    <string name="supportEmailId">support@eventyay.com</string>
+
+    <!--settings xml-->
+    <string name="version_settings">Version</string>
+    <string name="key_version">version</string>
+    <string name="about_settings">ABOUT</string>
+    <string name="key_rating">rating</string>
+    <string name="rating_settings">Rate Us</string>
+    <string name="app_name_capital">OPEN EVENT ANDROID</string>
+    <string name="suggestion_settings">Suggest Improvement</string>
+    <string name="key_suggestion">suggestion</string>
+    <string name="key_profile">profile</string>
+    <string name="profile_settings">PROFILE</string>
+    <string name="logged_in_account">Account</string>
+
+    <!--settings profile dialog-->
+    <string name="message">Are you sure you want to log out?</string>
+    <string name="cancel">Cancel</string>
+
+    <!--suggestion details-->
+    <string name="spread_sheet_link">https://docs.google.com/spreadsheets/d/1SzR75MBEVrTY1sDM3KAMm9wltiulDAp0QT5hv9eJkKM/edit#gid=1676755229</string>
+    <string name="suggestion_form_link">https://docs.google.com/forms/d/e/1FAIpQLSd7Y1T1xoXeYaAG_b6Tu1YYK-jZssoC5ltmQbkUX0kmDZaKYw/viewform</string>
```

Comment:
Try not to use strings.xml for links. unneeded indirection

Reply:
@iamareebjamal The spreadsheet link is not being used in the app, but as the sheet collects the responses, it is required. Should I comment the link in SettingsFragment?

Reply:
why don't you add the link in the readme

Reply:
Great! I will do that :+1: